project/Rat.scala

```diff
@@ -34,7 +34,9 @@ object Rat {
     file("images/daffodil.ico"),
     // yarn and rpm generated files
     file("yarn.lock"),
-    file("package-lock.json")
+    file("package-lock.json"),
+    // JAXB Bindings files
+    file("bindings.xjb")
```

Comment:
If a file can have a license, it should have one. An xjb file is just XML and so can have a license. This should be removed from the Rat excludes.